test/unit/tracks/text-track-settings.test.js

```diff
@@ -9,6 +9,16 @@ const tracks = [{
   label: 'test'
 }];
 
+const defaultSettings = {
```

Comment:
Yeah, I meant with how the workflow from this dialog is. The selection boxes turn into an object which we then use down the line to see if we need to apply extra styles to the elements that vttjs produced.
By default, no such extra processing should be done; also, if the persistence option is set, we shouldn't record these default values.
Checkboxes that toggle each override could be ok, since that's sort of what the `---` option is supposed to represent.